docs/data/date-pickers/date-picker/examplesConfig.styling.tsx

```diff
@@ -290,6 +290,30 @@ export const datePickerExamples: PickersSubcomponentType = {
     slots: ['root'],
     moreInformation: <TextFieldMoreInfo />,
   },
+  PickersPopper: {
+    examples: {
+      customTheme: {
+        type: 'success',
+        parentSlot: 'paper',
+      },
+      sxProp: {
+        type: 'warning',
+        parentSlot: 'popper',
+        current: true,
+        comments:
+          'Because of the structure of the DesktopDatePicker, the `sx` prop needs to be applied to the `popper` slot',
+      },
+      styledComponents: {
+        type: 'warning',
+        parentSlot: 'desktopPaper',
+        parentComponent: 'Paper',
```

Comment:
`PickersPopper` is an `internal` component, I think that it is better to show the compromise of overriding it with `MuiPaper` instead. 🤔